addons/website/static/src/snippets/s_process_steps/options.js

```diff
@@ -72,6 +72,17 @@ options.registry.StepsConnector = options.Class.extend({
     // Private
     //--------------------------------------------------------------------------
 
+    /**
+     * @override
+     */
+    _computeVisibility() {
+        // We don't use the service_context_get intentionally because the
+        // connectors are hidden as soon as the page is smaller than 992px
+        // (the BS lg breakpoint) and we don't use config.device.SIZES because
+        // it check the total window width and not only the iframe.
+        const isSmallDevice = this.$target[0].clientWidth <= 992;
```

Comment:
The comparison should not be made between the target width and 992, but rather between the window width.
e.g. with a "framed" page layout (the option is hidden at certain breakpoints, but it should not be.)
![Capture d’écran de 2023-03-06 09-56-50](https://user-images.githubusercontent.com/52911687/223064130-8a840a4e-e7b1-468e-8595-496bf6e55888.png)

Reply:
Also you should be able to obtain "992" by using `MEDIAS_BREAKPOINTS[SIZES.LG].minWidth;` instead of hardcoding it. Could check if it is possible ?